src/stpipe/cli/command.py

```diff
@@ -27,7 +29,8 @@ def add_subparser(cls, subparsers):
         subparsers : argparse._SubParsersAction
         """
 
-    @abc.abstractclassmethod
+    @abc.classmethod
```

Comment:
```suggestion
    @classmethod
```

Reply:
Done in https://github.com/spacetelescope/stpipe/commit/cef32eab5161f03c07dea93d67ce244ef69b7de3